### ISSUE TITLE
Flameshot gui not working after saving a screenshot as a file.

### ISSUE THREAD

**Issue by adamurban98** [lupoDharkael/flameshot#549]
**Flameshot version:** "Flameshot v0.6.0 Compiled with Qt 5.5.1"

**Describe the bug**  Flameshot gui command and Take Screenshots buttons not working  after using the save as a file option in the manual capture mode.

**To Reproduce** 
`flameshot gui`
click on diskette to save as a file
`flameshot gui` not working

The solution that works for me is to click Quit. And then restart Flameshot. But doing this after each saved screenshot is tedious.

**Expected behavior**

**System Information** Ubuntu 18.04.2 LTS

**Comment by LiterallyJohnny** [comment]
Do issues not get fixed around here? It's been over a year. I am also having the same issue.

**Comment by borgmanJeremy** [comment]
Yes issues are fixed.

**Comment by LiterallyJohnny** [comment]
Nope, I am having the same issues. Whenever I execute flameshot GUI and
attempt to save a screenshot, it just closes and nothing gets saved.
